### ISSUE TITLE
Fancy Up Next Buttons unresponsive, or unhighlighted.

### ISSUE THREAD

**Issue by mathu204** [im85288/service.upnext#236]
I even tried with the original upnext-upnext.xml but basically what happens is i press the developer option show a fancy pop up, and in the first image i cant see where the highlighted button is, i press any button to back out, go to simple, the highlight works as supposed to. If i press the simple and then go to fancy the fancy highlight will start working but only temporarily if i wait like 40 seconds its the same problem the only way i can fix it temporarily is by pressing simple, then going back to fancy. But I cant do this mid show, I thought maybe it was just a bug in the developer mode and it would work in the show, but that was not true same problem. I did press enter and it played the next episode tho so its just some sort of visual bug that it is not showing the highlighted colour or something like that here are some images. And a log.

[kodi.log](https://github.com/im85288/service.upnext/files/5682665/kodi.log)

https://imgur.com/a/VruegRF

**Comment by MoojMidge** [comment]
I quick look at the log seems to indicate that some textures are failing to load when the skin xml is being loaded to create the popup.

Not sure why this would happen, but being unfamiliar with the skin you are using, and to narrow things down would suggest testing using the default Kodi skin Estuary to see if the same issue occurs.

Perhaps the image files referenced in the xml don't exist in the skin or you have overwritten the xml in the skin, with the one from the addon or another skin that doesn't have the same directory structure?

**Comment by mathu204** [comment]
> I quick look at the log seems to indicate that some textures are failing to load when the skin xml is being loaded to create the popup.
> 
> Not sure why this would happen, but being unfamiliar with the skin you are using, and to narrow things down, would suggest testing using the default Kodi skin Estuary, as a first step to see if the same issue occurs.
> 
> Perhaps the image files referenced in the xml don't exist in the skin you are using, or you have overwritten the xml in the skin, with the one from the addon or another skin that doesn't have the same directory structure? Could also try reinstalling the skin you are using as a second step.

where would the new files/textures be located in a skin directory if you know? If it follows a default/set pathing across the board? thanks for the info i did just replace the skins integrated addon xml with the default one bc It did not look that nice to me

**Comment by MoojMidge** [comment]
The location of the image files would be dependent on the xml file.

If all you want to do is use the default popup appearance, rather than the one that comes with the skin, the easiest way is to just delete the xml files from the skin rather than copying the addon xml files.

**Comment by mathu204** [comment]
> The location of the image files would be dependent on the xml file.
> 
> If all you want to do is use the default popup appearance, rather than the one that comes with the skin, the easiest way is to just delete the xml files from the skin rather than copying the addon xml files.

Deleting the xml from the skin makes nothing pop up, copying the default addon xml and replacing the one in the skin brings me to here

**Comment by MoojMidge** [comment]
Are you reloading the skin after making the changes?

**Comment by mathu204** [comment]
> Are you reloading the skin after making the changes?

No I was not sorry I’m dumb let me try again

**Comment by mathu204** [comment]
That fixed it haha thank u :D
